package.json

```diff
@@ -17,7 +17,7 @@
     "example:start:aot": "yarn run cli serve --prod",
     "example:test": "jest -c projects/example-app/jest.config.js --watch",
     "example:build":"yarn cli build --prod",
-    "example:build:prod": "yarn cli build --prod --base-href \"/platform/example-app/\"",
+    "example:build:prod": "yarn run example:build -- --base-href \"/platform/example-app/\"",
```

Comment:
The `run` isn't needed here with yarn. In other words, this would become:

```
"example:build:prod": "yarn example:build -- --base-href \"/platform/example-app/\""
```